.github/workflows/ci.yml

```diff
@@ -15,9 +14,13 @@ concurrency:
   cancel-in-progress: ${{ (github.ref == 'refs/heads/staging') || (github.ref == 'refs/heads/trying') }}
 
 jobs:
-  test:
+  integration-tests:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 60
+    # We don't actually want to run integration tests on pull requests,
+    # because we want to avoid hitting rate limits.
+    # So, if this is a PR build, mark the integration tests as "skipped".
+    if: github.event_name != 'pull_request'
```

Comment:
Trying second option of https://github.com/JuliaLang/BumpStdlibs.jl/pull/110#discussion_r1903163462 (I can't do anything about the first one anyway)
```suggestion
```